### PR TITLE
Add DebuggingService and accounts method 

### PR DESCRIPTION
**Pull request by avive**
Required so we can add this capability to the api.

### CODE REVIEW

**Review by lrettig — CHANGES_REQUESTED** [review]
See question

File: proto/spacemesh/v1/global_state_types.proto [hunk 1]
```diff
@@ -31,6 +31,10 @@ message AccountResponse {
     Account account_wrapper = 1;
 }
 
+message AccountsResponse {
+    repeated Account account_wrapper = 1;
```
[the remark this answers is not in the document]
Comment (avive) [reply proto/spacemesh/v1/global_state_types.proto:35]: We can add a streaming method as well but I think that the pull one should be fine for now and solve the short term issue we have with testnet accounts and the issue with accounts in go-sm logs.

**Comment by avive** [comment]
Moved to DebuggingService.

**Review by lrettig — APPROVED** [review]
I gave the service a shorter name and added a note about not using the new service in production. LGTM.

**Pull request merged**